addons/hw_drivers/websocket_client.py

```diff
@@ -42,19 +43,30 @@ def send_to_controller(print_id, device_identifier):
 
 def on_message(ws, messages):
     """
-        When a message is receive, this function is triggered
-        The message is load and if its type is 'print', is sent to the printer
+        Synchronously handle messages received by the webscocket.
+        ! As it is synchronous, the operation must terminate quickly to avoid blocking the websocket.
```

Comment:
we don't need the comment 
"! As it is synchronous, the operation must terminate quickly to avoid blocking the websocket." in the method description as the "Synchronously handle" already covers it